src/web_service.iced

```diff
@@ -35,6 +35,12 @@ class WebServiceBinding extends Base
 
   #---------------
 
+  _type_v2 : (revoke_flag) ->
+    if @revoke? or revoke_flag then constants.sig_types_v2.web_service_binding_with_revoke
```

Comment:
actually it's a fair amount of work to make sibkey chainlinks from here. We'd catch an error like this on integration, since the thing wouldn't work.